web/src/main/java/com/epam/rft/atsy/web/controllers/FileUploadController.java

```diff
@@ -62,14 +62,15 @@ public ModelAndView uploadFile(@PathVariable("candidateId") Long candidateId,
 
     try {
       CandidateDTO candidateDTO = candidateService.getCandidate(candidateId);
-      if (candidateDTO.getCvFilename() != null && !candidateDTO.getCvFilename().isEmpty()) {
+      String candidateCVFilename = candidateDTO.getCvFilename();
+      if (candidateCVFilename != null && !candidateCVFilename.isEmpty()) {
```

Comment:
I would like to suggest this method: org.apache.commons.lang3.StringUtils.isNotEmpty